model/discord.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+import (
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
+)
+
+type DiscordUser struct {
+	ID        int    `db:"id"`
+	Email     string `db:"email"`
+	Token     string `db:"token"`
+	Confirmed bool   `db:"confirmed"`
+}
+
+type DiscordUserStatus string
+
+const (
+	NotFound  DiscordUserStatus = "not found"
+	Pending                     = "pending"
```

Comment:
You need to put `DiscordUserStatus` for `Pending` and `Confirmed`. Because an explicit value was provided, the type doesn't get copied from the previous line.